docker-compose.yml

```diff
@@ -9,14 +9,25 @@ services:
       # Only for dev
       - 3306:3306
     volumes:
-      # Persist data do ./mysql-data. Remove if you want to clear the db.
+      # Persist data to ./mysql-data. Remove if you want to clear the db.
       - ./mysql-data:/var/lib/mysql
     environment:
       MYSQL_DATABASE: starchart
       MYSQL_USER: starchart
       MYSQL_PASSWORD: starchart_password
       MYSQL_ROOT_PASSWORD: root_password
 
+  redis:
+    image: redis:7.0.8-alpine3.17
+    container_name: redis
+    command: ['redis-server']
+    ports:
+      # Only for dev
+      - 6379:6379
+    volumes:
+      # Persist data to ./redis-data. Remove if you want to clear the db.
+      - ./redis-data:/data
```

Comment:
So we are capable of persisting data on disk also with redis? (I always thought that redis is in-memory only)

Reply:
Yes.  You are correct that Redis is in-memory.  However, you have two options for persisting data:

1. append-only file
2. regular snapshots

Reply:
Great!